### PR TITLE
Tools page

### PR DESCRIPTION
**Pull request by chasulin**
Replaces `/resources` with `/tools` (with the same content, all now in the CMS, minus the SEIR model section). Applies styles/layout of Learn.

Todo:
- [x] Add embed
- [x] Move `/tools` to top level nav (?) and remove `/resources`
- [x] UX signoff

Note:
Products might no longer be a part of Learn altogether. Added in a TODO to move Products code + remove unused code if that is confirmed to be the case.

### CODE REVIEW

File: src/cms-content/learn/products-landing.json [hunk 1]
```diff
@@ -6,17 +6,26 @@
...
+    },
```
Comment (pnavarrc) [inline src/cms-content/learn/products-landing.json:17]: Note that there is `https://localhost:3000` here for a URL, we need to remove that prefix

File: src/App.js [hunk 2]
```diff
@@ -125,7 +121,7 @@ export default function App() {
               />
 
               <Route path="/about" component={About} />
-              <Route path="/resources" component={Resources} />
+              <Route path="/products" component={Products} />
```
Comment (pnavarrc) [inline src/App.js:124]: We need to support the `/resources` page as well in case we have links pointing to that or if users have bookmarks

```tsx
<Route path="/resources">
  <Redirect to="/products" />
</Route>
```

**Review by pnavarrc — CHANGES_REQUESTED** [review]
First pass!

File: src/cms-content/learn/products-landing.json (hunk 1 above)
Comment (chasulin) [reply src/cms-content/learn/products-landing.json:17]: oh whoops!! good catch. looks like copy/pasting the resources markdown into the cms took the localhost version of the links.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
-      - uses: actions/setup-node@v1.1.0
+      - uses: actions/setup-node@v1
```
Comment (chasulin) [inline .github/workflows/ci.yml:13]: replicates[ michael's fix](https://github.com/covid-projections/covid-projections/pull/1922)

File: src/screens/Embed/Embed.style.js [hunk 4]
```diff
@@ -156,7 +156,7 @@ export const EmbedBody = styled(Box)`
 export const EmbedHeaderWrapper = styled(Box)`
   ${PreviewHeaderStyles}
   width: 100%;
-  padding: 17px 14px;
+  padding: 13px 14px;
```
Comment (chasulin) [inline src/screens/Embed/Embed.style.js:159]: this edit is unrelated to this PR. i noticed that an update that was recently made to the embed to make the icons fix caused the top/bottom borders to get cut off (image below). this padding edit fixes that.

Screen Shot 2020-11-16 at 9 03 35 PM

File: src/cms-content/learn/learn-case-studies.json [hunk 5]
```diff
@@ -104,7 +104,7 @@
...
...ill the need for a powerful, localized tool with its **[COVID Response Simulator (CRS)](https://covidactnow.org/tools#covid-response-simulator)**. \n\nThe CRS allows users to view the potential impact of [non-pharmaceutical interventions](/glossary#npi) (NPIs), such as mask-wearing and closing schools, on COVID statistics like case numbers and hospitalizations. It includes CAN’s powerful epidemiology model and data pulled directly from official state and county sources, but all of the inputs and assumptions are modifiable and can be changed to reflect local realities. County officials can use this tool to predict the effects of different policy interventions and make informed decisions on how to manage COVID for their county’s specific circumstances.\n\n![Image of Covid Act Now's Covid Response Simulator, showing the location inputs, model assumptions, and graphs](/images_cms/naco-screenshot.png \"COVID Response Simulator\")\n\n> All of the inputs and assumptions are modifiable and can be changed to reflect local realities\n\nNACo features CAN’s CRS through its twice-weekly COVID update e-blasts to 40,000 county officials, and it has invited CAN to join its “Reopening Town Hall.”",
```
Comment (pnavarrc) [inline src/cms-content/learn/learn-case-studies.json:107]: Did this change or just the image?

File: src/cms-content/learn/products-landing.json [hunk 6]
```diff
@@ -1,24 +1,27 @@
...
...on your inputs, the simulator generates data and graphs illustrating COVID forecasts with and without these NPIs, including estimated case numbers and hospitalizations.\n\n![Example view of the COVID Response Simulator](/images_cms/crs-screenshot.png \"CRS\")\n\n> ### Get started\n>\n> 1. Before you begin, you can watch a full tutorial (30 minutes) on the simulator [here](https://youtu.be/6lYf9ry1DWE) or a shorter tutorial (10 minutes) on how to use the simulator specifically for modeling NPIs [here](https://youtu.be/xiI_kC8hGTA).\n> 2. Click on [this link](https://docs.google.com/spreadsheets/u/1/d/1iUQvuY-DGbuxGBDrj1fvaDjk-peunHm89sMJe0z24U8/copy?usp=sharing) to create a copy of the simulator for your use. You’ll be prompted to make a copy so that you can work in your own document.\n> 3. Review the information in the first tab in Orange labeled “About”. Then continue onward to the second tab in Green labeled “Location and Inputs Setup”.\n\n*If you have any questions about our model or the data we use, reach out to us at [info@covidactnow.org](mailto:info@covidactnow.org). You can also learn more on our [YouTube channel](https://www.youtube.com/channel/UChs298932BKWanjwWLVUO4w).*"
```
Comment (pnavarrc) [inline src/cms-content/learn/products-landing.json:16]: It's probably safer to link to `https://covidactnow.org/covid-response-simulator` here because it would be difficult to check that the link is up to date every time we update the CRS (we change the link each time)

File: src/screens/Tools/Tools.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+import { TocItem } from 'cms-content/utils';
+
+/**
+ * When updating the COVID_RESPONSE_SIMULATOR_URL, update the version number
+ * and the URL in src/assets/covid-response-simulator-version.txt. This file is
+ * used by the the CRS to determine if a copy is up to date, and show a
+ * message to the user.
+ */
+export const COVID_RESPONSE_SIMULATOR_URL =
+  'https://docs.google.com/spreadsheets/u/4/d/1vs-OjzBuE96sky-8CNJWLPWnXkJwsDKw5XBejYR5-uk/copy?usp=sharing';
```
Comment (pnavarrc) [inline src/screens/Tools/Tools.tsx:25]: I think this link is outdated, make sure to copy the correct one from develop before merging!

File: src/screens/Tools/Tools.tsx [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+export const sidebarSections: TocItem[] = [
+  {
+    label: 'Tools',
+    to: '/tools',
+    items: productsLandingContent.productsList.map(product => ({
+      to: `/tools#${product.productId}`,
+      label: product.productName,
+    })),
+  },
+];
```
Comment (pnavarrc) [inline src/screens/Tools/Tools.tsx:43]: nit: Consider moving this to the CMS as we do for some Learn pages (not strong feelings about this, but I think it makes sense to centralize document structure there and just use the structure and content in screens).

**Review by pnavarrc — APPROVED** [review]
Approving but we need to change 2 things before merging:
- Updating the CRS link 
- Link to covidactnow.org/covid-response-simulator on the CMS, so the link is always up to date

Looks good otherwise!

File: src/cms-content/learn/learn-case-studies.json (hunk 5 above)
Comment (chasulin) [reply src/cms-content/learn/learn-case-studies.json:107]: no, it changed the CRS link from resources to tools

Screen Shot 2020-11-17 at 6 18 19 PM

File: src/screens/Tools/Tools.tsx (hunk 8 above)
Comment (chasulin) [reply src/screens/Tools/Tools.tsx:43]: per our chat, punting this to a larger consideration of where these URLs should live (along with learn TOC urls, breadcrumbs, etc)
